### DISCUSSION TITLE
余额消耗问题

### DISCUSSION THREAD

**Discussion by benabcben** [ZuodaoTech/everyone-can-use-english#723] [Q&A]
我充值了 1 美金，加了 openai 的 api 一直用的openai的api ，就转化了一个 3 分钟的音频用的付费的服务，然后余额就变成-1.7 美金了 ，是不是扣错了，要是正常的话，转化 3 分钟的音频生成字幕能要 2.7 刀 吗，这也太贵了把

**Comment by an-lee** [comment]
是只生成了一次吗？还是有多次重新生成，或者编辑过？

**Comment by benabcben** [comment]
用 openai  的 api 生成了没成功用了一次付费的，然后效果不好，我在跟读的那个界面用 openai 又生成了几次，然后他居然就-1.7
了😂

an-lee ***@***.***> 于2024年6月26日周三 20:57写道：

**Reply by an-lee** [reply]
我查了后台，是扣了 20 次相同的费用。现在想确定一下你是重复用 AzureAI 生成了这么多次（重新生成也会扣费），还是有别的 bug。

我先给你返 2.7 刀余额。

**Comment by benabcben** [comment]
我在跟读界面里点选openai 的 api重新生成了 2 次还是 3 次，我记不清了，20 次是肯定没有的，然后我的音频大概是 5
分钟左右吧，然后我就绝望的*****的换了个号😂，然后那 1 刀能反到我这个邮箱重新注册的账号吗😂，我没用那个号了，，，

an-lee ***@***.***> 于2024年6月27日周四 20:26写道：

**Reply by an-lee** [reply]
可以。留下你的邮箱，我给你返

**Comment by benabcben** [comment]
***@***.***

an-lee ***@***.***> 于2024年6月27日周四 22:38写道：

**Reply by an-lee** [reply]
这个邮箱都被打码了。

**Reply by benabcben** [reply]
![图片1](https://github.com/ZuodaoTech/everyone-can-use-english/assets/63357233/499063c8-4a8a-457e-947a-475180f71f57)

**Comment by benabcben** [comment]
***@***.***
看不到吗[image: 图片1.png]

an-lee ***@***.***> 于2024年6月30日周日 08:15写道：

**Comment by an-lee** [comment]
已经返了，2.7 刀。

你都用啥插件了？Email 全被打码了。

![image](https://github.com/ZuodaoTech/everyone-can-use-english/assets/22209056/638820a5-ea0d-48f7-a392-9ec94caee128)

**Comment by benabcben** [comment]
不知道哇 ，邮件是转到我gmail上了，我发过去也没提示我邮箱打码😂

an-lee ***@***.***> 于2024年7月8日周一 18:42写道：

**Comment by benabcben** [comment]
今天更新了下，就这样了，进不去啊，咋回事[image: CleanShot 2024-07-19 at ***@***.***

lei xia ***@***.***> 于2024年7月9日周二 11:26写道：
